src/main/java/core/basesyntax/Calculable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Calculable {
+    
+    double getArea();
```

Comment:
if interface calculates - lets keep that concept
```suggestion
    double calculateArea();
```